### PR TITLE
Version 4.4.4 into develop

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Expected 1 space after &quot;&lt;&lt;&quot;; 0 found | [Abstract.php:1506](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Aggregator/Record/Abstract.php#L1506)
:x: | Expected 1 space before &quot;&lt;&lt;&quot;; 0 found | [Abstract.php:1506](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Aggregator/Record/Abstract.php#L1506)
:x: | Expected 1 space after &quot;&lt;&lt;&quot;; 0 found | [Abstract.php:1506](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Aggregator/Record/Abstract.php#L1506)
:x: | Expected 1 space after &quot;&lt;&lt;&quot;; 0 found | [File_Importer_Events.php:204](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Importer/File_Importer_Events.php#L204)
:x: | Expected 1 space before &quot;&lt;&lt;&quot;; 0 found | [File_Importer_Events.php:204](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Importer/File_Importer_Events.php#L204)
:x: | Expected 1 space after &quot;&lt;&lt;&quot;; 0 found | [File_Importer_Events.php:204](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Importer/File_Importer_Events.php#L204)
:x: | Expected 1 space after &quot;&lt;&lt;&quot;; 0 found | [Main.php:34](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Main.php#L34)
:x: | Expected 1 space before &quot;&lt;&lt;&quot;; 0 found | [Main.php:34](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Main.php#L34)
:x: | Expected 1 space after &quot;&lt;&lt;&quot;; 0 found | [Main.php:34](https://github.com/moderntribe/the-events-calendar/blob/8855d9598b2a4499f645bf02ede724487804cdf9/src/Tribe/Main.php#L34)

> via jenkins/codesniffer-phpcs

File: package.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "name": "the-events-calendar",
+<<<<<<< HEAD
```
Comment (barryhughes) [inline package.json]: Needs resolution

File: readme.txt [hunk 2]
```diff
@@ -4,8 +4,13 @@ Contributors: ModernTribe, borkweb, zbtirrell, barry.hughes, bordoni, brianjesse
 Tags: events, calendar, event, venue, organizer, dates, date, google maps, conference, workshop, concert, meeting, seminar, summit, class, modern tribe, tribe, widget
 Donate link: http://m.tri.be/29
 Requires at least: 3.9
+<<<<<<< HEAD
```
Comment (barryhughes) [inline readme.txt]: Needs resolution here too

File: readme.txt [hunk 3]
```diff
@@ -316,9 +321,17 @@ Please see the changelog for the complete list of changes in this release. Remem
 
 == Changelog ==
 
+<<<<<<< HEAD
```
Comment (barryhughes) [inline readme.txt]: Unresolved conflict here also

File: readme.txt [hunk 4]
```diff
@@ -316,9 +321,17 @@ Please see the changelog for the complete list of changes in this release. Remem
...
+= [4.4.4] 2017-03-08 =
+
+* Fix - Avoid unnecessarily removing a callback from an action while inside the same action (improves PolyLang compatibility - props @Chouby) [73122]
+* Fix - Resolving issue where sites that overrode the wp-admin path would fail to successfully perform a Facebook OAuth with Event Aggregator [74687]
+* Tweak - Improve template loading for easier customization by developers and better support of the template hierarchy (props @QWp6t) [72842]
+=======
 = [4.5] TBD =
 
 * Tweak - Clean up direct calls to get events and use wrapper tribe_get_events() which has a hook for customizing
+>>>>>>> develop
```
Comment (barryhughes) [inline readme.txt]: And here

**Comment by bordoni** [comment]
_Resolved Conflicts_

File: readme.txt [hunk 5]
```diff
@@ -4,7 +4,7 @@ Contributors: ModernTribe, borkweb, zbtirrell, barry.hughes, bordoni, brianjesse
 Tags: events, calendar, event, venue, organizer, dates, date, google maps, conference, workshop, concert, meeting, seminar, summit, class, modern tribe, tribe, widget
 Donate link: http://m.tri.be/29
 Requires at least: 3.9
-Tested up to: 4.7.2
+Tested up to: 4.7
```
Comment (barryhughes) [inline readme.txt]: Should have caught this yesterday - also, it's easy to fix when we next deploy from this branch - but it looks like we retained the wrong version number here

**Pull request merged**